sys/vane/rver.hoon

```diff
@@ -675,6 +675,31 @@
   ::
   |=  [[our=@p eny=@ =duct now=@da scry=sley] state=server-state]
   |%
+  ::  +request-local: bypass authentication for local lens connections
+  ::
+  ++  request-local
+    |=  [secure=? =address =request:http]
+    ^-  [(list move) server-state]
+    ::
+    =/  act  [%app app=%lens]
+    =/  connection=outstanding-connection
+      [act [& secure address request] ~ 0]
+    ::
+    =.  connections.state
+      (~(put by connections.state) duct connection)
+    ::
+    :_  state
+    :_  ~
+    :^  duct  %pass  /run-app/[app.act]
+    ^-  note
+    :^  %g  %deal  [our our]
```

Comment:
I worry a bit about how this appears to be hardcoding everything to the %lens app, but I suppose that's fine and maintaining backwards compatibility.

Reply:
Right - this PR is just restoring the old lens functionality. I think the correct approach would be to make lens (and herb) make and handle generic http requests, using dynamically-bound routes, generalized across local and remote servers. Then the only difference between `%request` and `%request-local` would be that the latter is pre-authorized. But all that will take some work.